### ISSUE TITLE
contrail-status warns about k8s environment lack of pod/service name

### ISSUE THREAD

**Issue by dmzoneill** [Juniper/contrail-charms#105]
Cosmetic issue, most likely, this is docker based, no k8s pods.

```
root@juju-0a8009-default-14:/home/ubuntu# contrail-status 
Pod              Service         Original Name                          State       Status                         
                 redis           contrail-external-redis                running     Up 13 minutes                  
config           api             contrail-controller-config-api         running     Up 14 minutes                  
config           device-manager  contrail-controller-config-devicemgr   running     Up 14 minutes                  
config           nodemgr         contrail-nodemgr                       running     Up 1 second                    
config           schema          contrail-controller-config-schema      running     Up Less than a second          
config           svc-monitor     contrail-controller-config-svcmonitor  restarting  Restarting (1) 54 seconds ago  
config-database  cassandra       contrail-external-cassandra            running     Up 14 minutes                  
config-database  nodemgr         contrail-nodemgr                       running     Up 14 minutes                  
config-database  rabbitmq        contrail-external-rabbitmq             running     Up 14 minutes                  
config-database  zookeeper       contrail-external-zookeeper            running     Up 14 minutes                  
control          control         contrail-controller-control-control    running     Up 14 minutes                  
control          dns             contrail-controller-control-dns        running     Up 14 minutes                  
control          named           contrail-controller-control-named      running     Up 14 minutes                  
control          nodemgr         contrail-nodemgr                       running     Up About a minute              
webui            job             contrail-controller-webui-job          running     Up 13 minutes                  
webui            web             contrail-controller-webui-web          running     Up 13 minutes                  

WARNING: container with original name 'contrail-external-redis' have Pod or Service empty. Pod: '' / Service: 'redis'. Please pass NODE_TYPE with pod name to container's env

== Contrail control ==
control: initializing (No BGP configuration for self)
nodemgr: initializing
named: active
dns: active

== Contrail config-database ==
nodemgr: initializing (NTP state unsynchronized. )
zookeeper: active
rabbitmq: active
cassandra: active

== Contrail config ==
svc-monitor: inactive
nodemgr: initializing
device-manager: active
api: active
schema: backup

== Contrail webui ==
web: active
job: active

```

```
root@juju-0a8009-default-14:/home/ubuntu# docker images
REPOSITORY                                                       TAG                  IMAGE ID            CREATED             SIZE
hub.juniper.net/contrail/contrail-status                         5.0.2-0.360-queens   0d4c16bed89b        8 months ago        780MB
hub.juniper.net/contrail/contrail-nodemgr                        5.0.2-0.360-queens   f54f27748520        8 months ago        794MB
hub.juniper.net/contrail/contrail-node-init                      5.0.2-0.360-queens   84fdf6094a00        8 months ago        779MB
hub.juniper.net/contrail/contrail-external-zookeeper             5.0.2-0.360-queens   f1f44b12947b        8 months ago        144MB
hub.juniper.net/contrail/contrail-external-redis                 5.0.2-0.360-queens   5847046c6781        8 months ago        107MB
hub.juniper.net/contrail/contrail-external-rabbitmq              5.0.2-0.360-queens   21871e76bfe8        8 months ago        189MB
hub.juniper.net/contrail/contrail-external-cassandra             5.0.2-0.360-queens   872973896500        8 months ago        323MB
hub.juniper.net/contrail/contrail-controller-webui-web           5.0.2-0.360-queens   d3ece31ba7e5        8 months ago        998MB
hub.juniper.net/contrail/contrail-controller-webui-job           5.0.2-0.360-queens   8511cd2a827e        8 months ago        998MB
hub.juniper.net/contrail/contrail-controller-control-named       5.0.2-0.360-queens   7cd1d79561d2        8 months ago        847MB
hub.juniper.net/contrail/contrail-controller-control-dns         5.0.2-0.360-queens   1c5dbdf80f2b        8 months ago        847MB
hub.juniper.net/contrail/contrail-controller-control-control     5.0.2-0.360-queens   1603de590ccf        8 months ago        847MB
hub.juniper.net/contrail/contrail-controller-config-svcmonitor   5.0.2-0.360-queens   2029c140dbb4        8 months ago        1.07GB
hub.juniper.net/contrail/contrail-controller-config-schema       5.0.2-0.360-queens   e965cbf5d6dd        8 months ago        1.07GB
hub.juniper.net/contrail/contrail-controller-config-devicemgr    5.0.2-0.360-queens   0139d9b9bf6e        8 months ago        1.07GB
hub.juniper.net/contrail/contrail-controller-config-api          5.0.2-0.360-queens   7d065313f991        8 months ago        1.13GB
root@juju-0a8009-default-14:/home/ubuntu# 
```

**Comment by Andrey-mp** [comment]
this is an issue in 5.0.2 and was fixed in 5.1

**Comment by dmzoneill** [comment]
thank you
